css/index.css

```
@@ -74,7 +73,100 @@ body {
   justify-content: center;
 }
 
-.form-div {
+.mainPage {
+  height: 600px;
+  width: 380px;
+  background-color: #fff;
+  border: 1px solid #c7c7c7;
+  margin: 30px 200px 50px 520px;
+  position: center;
```

The `position` property is not being properly used, you should read about the property [here](https://developer.mozilla.org/en-US/docs/Web/CSS/position).
```suggestion
  margin: 30px auto 50px auto;
```